### PR TITLE
Create project Button

### LINKED ISSUE #27: Implementation of "Create Project" button on the projects page
## Description
There is a button currently present in the view, but it is not doing any action.

## Criteria
- [ ] Clicking the button should create a card in the list with default values
- [ ] Should also create the according folder

### PR DESCRIPTION
**Pull request by zelcion**
Closes #27

### CODE REVIEW

File: src/electron/main.ts
Comment (homemmakako) [inline src/electron/main.ts:30]: Nice! Had changed it as well on local branch after the change to auto open the dev tools

File: src/entities/controllers/editor-metadata-controller.ts
Comment (homemmakako) [inline src/entities/controllers/editor-metadata-controller.ts:20]: Maybe remove this before merge

File: src/entities/models/project.ts
Comment (homemmakako) [inline src/entities/models/project.ts:58]: Personally I would capitalize the first letter, but thats more of a stylistic choice

**Review by homemmakako — APPROVED** [review]
Some minor comments. Nothing that should block the PR. Good job!

File: src/entities/controllers/editor-metadata-controller.ts
Comment (zelcion) [reply src/entities/controllers/editor-metadata-controller.ts:20]: nice catch

**Pull request merged**
